io.github.dweymouth.supersonic.yml

```diff
@@ -197,6 +197,8 @@ modules:
       - . /usr/lib/sdk/golang/enable.sh && CGO_LDFLAGS=-L/app/lib go build -v .
       - . /usr/lib/sdk/golang/enable.sh && CGO_LDFLAGS=-L/app/lib /app/bin/fyne package -os linux -name Supersonic -icon ./res/appicon-500.png
       - tar -C /app --strip-components=1 -x -v -f /run/build/supersonic/Supersonic.tar.xz
+      - mkdir -p /app/share/icons
+      - mv /app/share/pixmaps/Supersonic.png /app/share/icons
```

Comment:
it should be `/app/share/icons/hicolor/SIZExSIZE/apps`

with `SIZE` being the size of the icon.

Reply:
i tried this:

```
modified   io.github.dweymouth.supersonic.yml
@@ -197,8 +197,8 @@ modules:
       - . /usr/lib/sdk/golang/enable.sh && CGO_LDFLAGS=-L/app/lib go build -v .
       - . /usr/lib/sdk/golang/enable.sh && CGO_LDFLAGS=-L/app/lib /app/bin/fyne package -os linux -name Supersonic -icon ./res/appicon-500.png
       - tar -C /app --strip-components=1 -x -v -f /run/build/supersonic/Supersonic.tar.xz
-      - mkdir -p /app/share/icons
-      - mv /app/share/pixmaps/Supersonic.png /app/share/icons
+      - mkdir -p /app/share/icons/hicolor/520x520/
+      - mv /app/share/pixmaps/Supersonic.png /app/share/icons/hicolor/520x520/
     sources:
       - type: git
         url: https://github.com/dweymouth/supersonic/
```

and still got this error: 

```
Error: icon Supersonic not found below /home/anarcat/src/flathub/.flatpak-builder/rofiles/rofiles-qg3xRa/files/share/icons
```

is there a standard set of sizes the icon should live in?

Reply:
520x520 is not a standard size. also it's 512 maximum.

Reply:
i filed https://github.com/dweymouth/supersonic/issues/132 upstream. i have tried to throw the 520x520 icon in the 512x512 directory and that still doesn'T work... i am beginning to suspect the rename call is what is broken here, unless it's actually checking the file resolution?

Reply:
also it's still the wrong path. missing `apps` as the last directory.

Reply:
it is, and i fixed this.